package.json

```diff
@@ -12,8 +12,12 @@
     "build-storybook": "storybook build"
   },
   "dependencies": {
+    "localforage": "^1.10.0",
+    "match-sorter": "^6.3.3",
```

Comment:
Correct me if I'm wrong, but I think these packages are not being used, also the `sort-by` package. Kindly remove them if unused. Use npm uninstall :bow: